### DISCUSSION TITLE
Today's video upload

### DISCUSSION THREAD

**Discussion by sai-pranay-teja** [learndevopsonline/learndevopsonline#734] [Q&A]
Hi @r-devops 

Please upload today's video session

Thank you

**Comment by rajace7** [comment]
today no class

![image](https://user-images.githubusercontent.com/129022778/234926528-263240ba-e216-44a4-a0c3-4c812f361781.png)

**Reply by sai-pranay-teja** [reply]
Hi @rajace7 


Today's video link:
https://www.youtube.com/watch?v=nx4Z4-LC4-o&list=PLLmNg_q93mewVIHleI2YGrRUmt6-ApK3E&index=71

Thanks and regards
